e2e/test/scenarios/native-filters/sql-field-filter.cy.spec.js

```diff
@@ -53,16 +53,6 @@ describe("scenarios > filters > sql filters > field filter", () => {
         cy.findByText("Small Marble Shoes");
         cy.findByText("Rustic Paper Wallet");
       });
-
-      cy.log("set the value through the filter widget");
```

Comment:
@nemanjaglumac  Same here, this checked if a required parameter with unset default value could be set through the filter widget — but now it's impossible, and I am removing this check.

NB: I haven't added new e2e tests covering the new behavior yet.